provider/lxd/credentials_test.go

```diff
@@ -286,7 +291,9 @@ func (s *credentialsSuite) TestRemoteDetectCredentialsWithCertFailure(c *gc.C) {
 	deps := s.createProvider(ctrl)
 	s.setupLocalhost(deps, c)
 
-	deps.configReader.EXPECT().ReadConfig(".config/lxc/config.yml").Return(lxd.LXCConfig{
+	deps.configReader.EXPECT().ReadConfig(path.Join(osenv.JujuXDGDataHomePath("lxd"), "config.yml")).Return(lxd.LXCConfig{}, nil)
```

Comment:
Could make this block a method on `credentialsSuiteDeps` that accepts the return.